config/src/shortcuts/mod.rs

```diff
@@ -49,6 +52,31 @@ pub fn shortcuts(context: &cosmic_config::Config) -> Shortcuts {
     shortcuts
 }
 
+/// Get the current system gesture configuration
+///
+/// Merges user-defined custom gestures to the system default config
+pub fn gestures(context: &cosmic_config::Config) -> Gestures {
+    // Load gestures defined by the system.
+    let mut gestures = context
+        .get::<Gestures>("default_gestures")
+        .unwrap_or_else(|why| {
+            tracing::error!("shortcuts defaults config error: {why:?}");
```

Comment:
Use `if why.is_err() {}` to check if the error is worth logging. Otherwise this wills pam the logs if the config is not in use.